### DISCUSSION TITLE
Redmi Note 8 miui 12.5 Android 11

### DISCUSSION THREAD

**Discussion by LukeOkk** [skyline-emu/skyline#355] [Ideas]
![Screenshot_2022-01-30-01-52-12-794_com nianticlabs pokemongo](https://user-images.githubusercontent.com/85941941/151687130-a139ebc1-da07-4a01-8c09-8862034481e7.jpg)
![Screenshot_2022-01-30-01-48-18-841_skyline emu](https://user-images.githubusercontent.com/85941941/151687131-e1344b42-9065-42d7-ad9d-d963ce7ea4f3.jpg)

FPS low, add new option drivers hardware compatibility frecuency in android please.

**Comment by Bossplay228** [comment]
У меня redmi note 10 pro но fps тоже маленький 7 - 10 Fps

**Reply by LukeOkk** [reply]
egg ns достигает 60 кадров в секунду на наших телефонах, это связано с частотой и совместимостью приложения с Android. не эмулятор

**Comment by Allatel** [comment]
Dood. There was no graphics a month ago. Ofc there will be bad fps, especially on non flagship models.
